### DISCUSSION TITLE
hello

### DISCUSSION THREAD

**Discussion by hosseincharchianh** [community/community#130282] [New to GitHub]
hello

**Comment by alexisseurin** [comment]
Welcome to GitHub :)

**Reply by hosseincharchianh** [reply]
thanks <3

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @hosseincharchianh 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
